### ISSUE TITLE
Typo in application.py, .batch tries to call app.bath

### ISSUE THREAD

**Issue by Alyxion** [loonghao/photoshop-python-api#204]
```
    def batch(self, *args, **kwargs):
        """Runs the batch automation routine.
        Similar to the **File** > **Automate** > **Batch** command.
        """
        self.app.bath(*args, **kwargs)
```

The batch function tries to call self.app.**bath** which doesn't exist.

As I assume it neither wants to take a bath nor a shower I assume it's intended to call `batch`.

**Comment by loonghao** [comment]
@Alyxion Thanks for the report.

I double checked we also miss implement `BatchOptions`, 

`app.batch` needs `BatchOptions`
https://community.adobe.com/t5/photoshop-ecosystem-discussions/how-to-run-photoshop-batch-via-javascript-vs-manually-selecting-all-attributes/m-p/12474797


I will find time to update and fix them together

**Comment by loonghao** [comment]
@Alyxion I released a new version to implement `app.batch` and fix typo

have a new example for this：
https://loonghao.github.io/photoshop-python-api/examples/#run-batch

**Comment by loonghao** [comment]
@allcontributors please add @Alyxion for Bug reports.

**Comment by Alyxion** [comment]
Awesome, thank you very much for the fix :).
Will give a try as soon as I've finished my current project. As I needed it for a project (your package helped me here a lot, so thanks!) I had to build a workaround meanwhile.
One evil thing in this work around was - though I assume you will have the same issue with batch processing because it seems to don't throw a classic com exception -  that "Quickactions -> Remove background" opens a Dialog window if you provide a picture which is very blurry or really doesn't contain an object. All batch processes actually just completely halted then waited for someone to click the "OK". In the end I ended up writing a background thread which watches if the main thread is stuck and if so kills Photoshop via 
```
for proc in psutil.process_iter():
        if proc.name().lower() == "photoshop.exe":
            print("Photoshop timed out, trying to kill Photoshop...")
            proc.kill()
            time.sleep(2.0)
            return True
    return False
```
Hope Adobe will fix this soon.
